MobileWallet/Common/BackgroundTasks/ReminderNotifications.swift

```diff
@@ -64,15 +64,15 @@ class ReminderNotifications {
         ScheduledReminder(
             identifier: "scheduled-reminders-recipient-1",
             title: ReminderNotifications.titleString,
-            body: NSLocalizedString("Open Tari Aurora", comment: "Local reminder notifications"),
+            body: NSLocalizedString("short-body-scheduled-reminder", comment: "Local reminder notifications"),
             deliverAfter: 60 * 60 * 24
         ),
         ScheduledReminder(
             identifier: "scheduled-reminders-recipient-2",
             title: ReminderNotifications.titleString,
             body: String(
                 format: NSLocalizedString(
-                    "Someone sent you %@! Open Tari Aurora to receive it now or it will be returned to the sender.",
+                    "long-body-scheduled-reminder %@",
```

Comment:
You can remove the %@ from the key here. That's a placeholder var, it only needs to be in the value string

Reply:
i know, this is to make it clear that there is a variable

Reply:
Ok cool 👍

Reply:
`scheduled_reminder_body_long`

Reply:
@kukabi without "%@"?

Reply:
What we decided about %@? It's have not influence to string but helps to visual mark localizable string with variable.

Reply:
Let's wait for @Jasonvdb 's comment after I finish my review. @SerhiiShovkoplias

Reply:
`scheduled_reminder.body.long`